CHANGES.rst

```diff
@@ -1,3 +1,564 @@
+
+- Moved angle formatting and parsing utilities to
+  ``astropy.coordinates.angle_formats``.
+  Added new functionality to ``astropy.coordinates.angle_utilities`` for
+  generating points on or in spherical surfaces, either randomly or on a grid. [#11628]
+
+- Added a new method to ``SkyCoord``, ``spherical_offsets_by()``, which is the
+  conceptual inverse of ``spherical_offsets_to()``: Given angular offsets in
+  longitude and latitude, this method returns a new coordinate with the offsets
+  applied. [#11635]
+
+astropy.io.ascii
+^^^^^^^^^^^^^^^^
+
+- Added support for reading and writing ASCII tables in QDP (Quick and Dandy Plotter) format. [#11256]
+
+- Added support for reading and writing multidimensional column data (masked and
+  unmasked) to ECSV. Also added formal support for reading and writing object-type
+  column data which can contain items consisting of lists, dicts, and basic scalar
+  types. This can be used to store columns of variable-length arrays. Both of
+  these features use JSON to convert the object to a string that is stored in the
+  ECSV output. [#11569, #11662, #11720]
+
+astropy.io.fits
+^^^^^^^^^^^^^^^
+
+- Added ``append`` keyword to append table objects to an existing FITS file [#2632, #11149]
+
+- Check that the SIMPLE card is present when opening a file, to ensure that the
+  file is a valid FITS file and raise a better error when opening a non FITS
+  one. ``ignore_missing_simple`` can be used to skip this verification. [#10895]
+
+- Expose ``Header.strip`` as a public method, to remove the most common
+  structural keywords. [#11174]
+
+- Enable the use of ``os.PathLike`` objects when dealing with (mainly FITS) files. [#11580]
+
+astropy.io.registry
+^^^^^^^^^^^^^^^^^^^
+
+- Readers and writers can now set a priority, to assist with resolving which
+  format to use. [#11214]
+
+astropy.io.votable
+^^^^^^^^^^^^^^^^^^
+
+- Version 1.4 VOTables now use the VOUnit format specification. [#11032]
+
+- When reading VOTables using the Unified File Read/Write Interface (i.e. using
+  the ``Table.read()`` or ``QTable.read()`` functions) it is now possible to
+  specify all keyword arguments that are valid for
+  ``astropy.io.votable.table.parse()``. [#11643]
+
+astropy.modeling
+^^^^^^^^^^^^^^^^
+
+- Added a state attribute to models to allow preventing the synching of
+  constraint values from the constituent models. This synching can
+  greatly slow down fitting if there are large numbers of fit parameters.
+  model.sync_constraints = True means check constituent model constraints
+  for compound models every time the constraint is accessed, False, do not.
+  Fitters that support constraints will set this to False on the model copy
+  and then set back to True when the fit is complete before returning. [#11365]
+
+- The ``convolve_models_fft`` function implements model convolution so that one
+- insures that the convolution remains consistent across multiple different
+- inputs. [#11456]
```

Comment:
```suggestion
- The ``convolve_models_fft`` function implements model convolution so that one
 insures that the convolution remains consistent across multiple different
 inputs. [#11456]
```